src/components/o-testimonials/index.scss

```diff
@@ -0,0 +1,147 @@
+$o-testimonials-subtitle-margin-top-xs: 20px;
+$o-testimonials-content-margin-xs: 10px;
+$o-testimonials-content-margin-sm: 40px;
+$o-testimonials-navigator-margin-horizontal-xs: 20px;
+$o-testimonials-navigator-margin-horizontal-sm: 30px;
+$o-testimonials-navigator-margin-horizontal-lg: 90px;
+
+$o-testimonials-arrow-width-xs: 8px;
+$o-testimonials-arrow-height-xs: 16px;
+$o-testimonials-arrow-width-sm: 15px;
+$o-testimonials-arrow-height-sm: 30px;
+$o-testimonials-arrow-border: 2px;
+
+.o-testimonials__background {
+  display: inline-block;
+  width: 100%;
+  background: $color-sec-green-viridian;
+  color: $color-prim-white;
+}
+
+.o-testimonials__title {
+  @include typo-small-module-title--publico();
+  margin: $o-testimonials-title-margin-top-xs $o-testimonials-margin-30 0 $o-testimonials-margin-30;
+  text-align: center;
+
+  @include respond-up(sm) {
+    margin: $o-testimonials-title-margin-top-sm $o-testimonials-margin-30 0 $o-testimonials-margin-30;
+  }
+}
+
+.o-testimonials__subtitle {
+  @include word-break();
+  @include typo-tag();
+
+  margin: $o-testimonials-subtitle-margin-top-xs $o-testimonials-margin-30 0 $o-testimonials-margin-30;
+  text-align: center;
+}
+
+.o-testimonials__content {
+  position: relative;
+
+  width: 100%;
+  margin: $o-testimonials-content-margin-xs;
+  overflow: hidden;
+
+  @include respond-up(sm) {
+    margin: $o-testimonials-content-margin-sm $o-testimonials-margin-30 $o-testimonials-content-margin-sm $o-testimonials-margin-30;
+  }
+}
+
+.o-testimonials__arrow-left {
+  padding: $o-testimonials-arrow-width-xs;
+  height: $o-testimonials-arrow-height-xs;
+  width: $o-testimonials-arrow-width-xs;
+  border: solid $color-prim-white;
+  border-width: $o-testimonials-arrow-border 0 0 $o-testimonials-arrow-border;
+  transform: rotate(-45deg);
+  cursor: pointer;
+
+  @include respond-up(sm) {
+    padding: $o-testimonials-arrow-width-sm;
+    height: $o-testimonials-arrow-height-sm;
+    width: $o-testimonials-arrow-width-sm;
+  }
+}
+
+.o-testimonials__arrow-right {
+  height: $o-testimonials-arrow-height-xs;
+  width: $o-testimonials-arrow-width-xs;
+  padding: $o-testimonials-arrow-width-xs;
+  border: solid $color-prim-white;
+  border-width: $o-testimonials-arrow-border 0 0 $o-testimonials-arrow-border;
+  transform: rotate(135deg);
+  cursor: pointer;
+
+  @include respond-up(sm) {
+    padding: $o-testimonials-arrow-width-sm;
+    height: $o-testimonials-arrow-height-sm;
+    width: $o-testimonials-arrow-width-sm;
+  }
+}
+
+.o-testimonials__navigator {
+  display: flex;
+
+  margin: 0 $o-testimonials-navigator-margin-horizontal-xs;
+
+  flex-direction: row;
+  align-items: center;
+
+  @include respond-up(sm) {
+    margin: 0 $o-testimonials-navigator-margin-horizontal-sm;
+  }
+
+  @include respond-up(lg) {
+    margin: 0 $o-testimonials-navigator-margin-horizontal-lg;
+  }
+}
+
+.o-testimonials__item {
+  @include word-break();
+  @include typo-tag();
+  text-align: center;
```

Comment:
newline before missing

Reply:
Travis is ok with current Version

Reply:
then its ok for me

Reply:
Unfortunately our stylelint config is missing to check for newlines between `@at-rules`, declarations, selectors etc.
I created a bug #545

Reply:
I dont think that lines will bring a better quality code.. important is that the order of the groups are respected. We can discuss it